src/directives/price.ts

```diff
@@ -11,28 +12,23 @@ import formatPrice from '@/helpers/format-price';
  * Displaying with currency symbol behind:
  * <span v-price.currencyBefore="1000"></span> => CHF 10.00
  * <span v-price.currencyAfter="1000"></span> => 10.00 CHF
- *
- * @param {Object} options
- * @param {Boolean} options.value - The price value in cents.
- * @param {Boolean} [options.currencyBefore] - The modifier that renders the currency before the price.
- * @param {Boolean} [options.currencyAfter] - The modifier that renders the currency after the price.
  */
 export default {
   name: 'price',
 
-  bind(el, binding) {
+  beforeMount(el: HTMLElement, binding: DirectiveBinding) {
```

Comment:
Please extract the callback function as separate method and assign it to `beforeMount` and `updated` to reduce duplicated code.